mysql2.gemspec

```diff
@@ -24,4 +24,5 @@ Mysql2::GEMSPEC = Gem::Specification.new do |s|
   s.metadata['msys2_mingw_dependencies'] = 'libmariadbclient'
 
   s.add_runtime_dependency 'bigdecimal'
+  s.add_runtime_dependency 'aws-sdk-rds'
```

Comment:
This adds aws-sdk dependency to everyone. You can make it optional dependency (not specify in gemspec and raise if require fails with message like please add this gem to your bundle to use this functionality).

Reply:
@simi thank you for the quick feedback. I have made this change.